tests/network/connectivity/test_pod_network.py

```diff
@@ -61,12 +62,18 @@ def pod_net_vmb(
 
 @pytest.fixture()
 def pod_net_running_vma(pod_net_vma):
-    return running_vm(vm=pod_net_vma, wait_for_cloud_init=True)
+    pod_net_vma.vmi.wait_for_condition(
```

Comment:
Why not use wait_for_guest_agent function?
```suggestion
    wait_for_guest_agent(vmi=pod_net_vma.vmi)
```

Reply:
Because `wait_for_guest_agent` is a duplicate of an existing mechanism available in the wrapper.
Same answer for all other places.

We could add a method to the VM object to abstract the two details here, but it seems a nit to me. If someone which needs to maintain this codebase think it is worth it, then fine by me.